.gitignore

```diff
@@ -16,3 +16,4 @@ misc/test-ca/demoCA/newcerts/*.pem
 tmp/
 /doc/workdir/
 include/h2o/gitrev.h
+/_build/
```

Comment:
Does this need to start from an underscore? I ask this because we already have “build/“ in the list.

If having underscore is your preference (or a custom in certain communities), how about adding /*build/ or something to merge the two names?

Reply:
actually I will remove this; I use this all the time for erlang & elixir projects but there's no need to leak into h2o.